scheduler/src/cook/kubernetes/api.clj

```diff
@@ -1641,6 +1664,13 @@
                 (str "Pod name from pod (" pod-name-from-pod ") "
                      "does not match pod name argument (" pod-name ")"))
         (log/info "In" compute-cluster-name "compute cluster, launching pod with name" pod-name "in namespace" namespace ":" (.serialize json pod))
+        (let [event-map (merge
+                          {:cluster-name compute-cluster-name
+                           :event-type passport/pod-launching
+                           :namespace namespace
+                           :pod-name pod-name}
+                          (create-uuid-map pod-name))]
```

Comment:
Maybe to avoid the `merge`, you could call the helper function something like `assoc-uuid` and have it take in the map to `assoc` onto as an argument?

Reply:
That makes sense, I can use `cond->` as before too.